doc/source/whatsnew/v0.23.0.txt

```diff
@@ -1375,6 +1375,7 @@ Reshaping
 - Bug in :func:`isna`, which cannot handle ambiguous typed lists (:issue:`20675`)
 - Bug in :func:`concat` which raises an error when concatenating TZ-aware dataframes and all-NaT dataframes (:issue:`12396`)
 - Bug in :func:`concat` which raises an error when concatenating empty TZ-aware series (:issue:`18447`)
+- Updated :func:`~pandas.pivot_table` with more comprehensive examples. Also updated Reshaping and Pivot Tables documentation with a Frequenty Asked Questions example (:issue:`19089`)
```

Comment:
We don't typically add a whatsnew note for documentation-only updates so you can remove this